docs/manage/users-remote.rst

```diff
@@ -0,0 +1,154 @@
+.. _remote-users:
+
+#######################
+ Managing Remote Users
+#######################
+
+Determined lets you manage users and user provisioning remotely. Remote user provisioning lets you
+include and synchronize any information about the user stored in your IdP such as their username,
+display name, and groups. Once configured, you can manage remote users without having to manually
+modify the users or update them via SCIM. Each time the remote user accesses Determined, their
+information is synchronized.
+
+.. include:: ../_shared/attn-enterprise-edition.txt
+
+.. note::
+
+   As of 026.4, OIDC is supported.
+
+*******************************
+ Enable Remote User Management
+*******************************
+
+Set the Auto Provision Option
+=============================
+
+The first step is to configure the :ref:`master configuration file <master-config-reference>` to
+enable auto provisioning users and the remote management of any information attached to the users.
+
+.. tabs::
+
+   .. tab::
+
+      OIDC
+
+      -  Enable OIDC.
+      -  Set ``oidc.auto_provision_users`` option to ``true`` in your :ref:`master configuration
+         file <master-config-reference>`.
+
+      .. note::
+
+         If ``scim_enabled`` is ``true``, then ``oidc.auto_provision_users`` must be ``false``.
+
+      For example:
+
+      .. code:: yaml
+
+         oidc:
+             enabled: true
+             provider: "Okta"
+             idp_recipient_url: "https://determined.example.com/oidc/sso"
+             idp_sso_url: "https://dev-00000000.okta.com"
+             client_id: "xx0xx0"
+             client_secret: "xx0xx0"
+             auto_provision_users: true
+             display_name_claim_name: "XYZ"
+
+   .. tab::
+
+      SAML
+
```

Comment:
@salonig23 SAML section for review. if the info is correct, i will either leave it in the PR or move the content to its own PR depending on timing.